vector-im/element-android
# [Live location sharing] - Removing BeaconInfo structure

## Type of change


## Content


Updating the beacon state event content sent during a live location share. Some fields are no longer encapsulated into an extra data layer.
Fixing state event content sent when stopping a live: only the "live" field is updated.

## Motivation and context


See [MSC-3672](https://github.com/matrix-org/matrix-spec-proposals/blob/stefan/ephemeral-location-streaming/proposals/3672-ephemeral-location-streaming.md#proposal)

## Screenshots / GIFs





## Tests



Ensure to enable location sharing in Preferences and the live location feature flag in developer settings.

- Start a live location share into a room
- Check you see the start message of the live

Note: other live status are not currently handled (e.g. location data, stopped)

## Tested devices

- OS version(s): 11

## Checklist
